Packages/MIES/MIES_PulseAveraging.ipf

```diff
@@ -1628,7 +1628,7 @@ static Function PA_ZeroPulses(WAVE/WAVE set, STRUCT PulseAverageSettings &pa)
 	endif
 
 	Make/FREE/N=(DimSize(set, ROWS)) junkWave
-	MultiThread junkWave = ZeroWave(set[p]) + PA_UpdateMinAndMax(set[p])
+	MultiThread junkWave = ZeroWave(set[p]) && PA_UpdateMinAndMax(set[p])
```

Comment:
@t-b can we talk about this line the next time we chat?

Reply:
@timjarsky If zeroWave(set[p]) returns 1, then PA_UpdateMinAndMax(set[p]) is run